topi/src/topi.cc

```diff
@@ -595,7 +602,7 @@ TVM_REGISTER_GLOBAL("topi.generic.schedule_injective")
 TVM_REGISTER_GLOBAL("topi.generic.schedule_injective_from_existing")
 .set_body([](TVMArgs args, TVMRetValue *rv) {
   *rv = topi::generic::schedule_injective_from_existing(args[0], args[1]);
- });
+  });
```

Comment:
Remove this diff

Reply:
I got following lint error for those three lines,
```
topi/src/topi.cc:605:  Weird number of spaces at line-start.  Are you using a 2-space indent?  [whitespace/indent] [3]
```
 and I found all other similiar functions have correct indent, so I changed them: